arraycontext/loopy.py

```diff
@@ -68,6 +70,80 @@ def get_default_entrypoint(t_unit):
             raise TypeError("unable to find default entry point for loopy "
                     "translation unit")
 
+
+def _get_scalar_func_loopy_program(actx, c_name, nargs, naxes):
```

Comment:
A recent merge #122 has changed the implementation here, could you please use the latest function definition from the trunk.